export/dataProvider/dataReader/Proctor.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019  (original work) Open Assessment Technologies SA;
+ *
+ * @author Yuri Filippovich
+ */
+
+namespace oat\taoSyncServer\export\dataProvider\dataReader;
+
+use oat\generis\model\GenerisRdf;
+use oat\tao\model\TaoOntology;
+use oat\taoProctoring\model\ProctorService;
+use oat\taoSync\export\dataProvider\dataReader\AbstractDataReader;
+use oat\taoSyncServer\exception\DataProviderException;
+use oat\taoTestCenter\model\ProctorManagementService;
+
+class Proctor extends AbstractDataReader
```

Comment:
SyncProctorDataReader

Reply:
I think we don't need to create classes with huge names as we have namespaces

Reply:
but 1. You don't see the namespace in the tab when the file is open, so you need to scroll to the namespace declaration to be sure and 2. you can't find quickly your class with full-text search